keystore/src/connection/platform/wasm/mod.rs

```diff
@@ -42,9 +42,9 @@ impl DatabaseConnection for WasmConnection {
         let name = name.to_string();
         // ? Maybe find a cleaner way to define the schema
 
-        let version_major = u32::from_str_radix(env!("CARGO_PKG_VERSION_MAJOR"), 10).unwrap_or_default();
-        let version_minor = u32::from_str_radix(env!("CARGO_PKG_VERSION_MINOR"), 10).unwrap_or_default();
-        let version_patch = u32::from_str_radix(env!("CARGO_PKG_VERSION_PATCH"), 10).unwrap_or_default();
+        let version_major = env!("CARGO_PKG_VERSION_MAJOR").parse::<u32>()?;
+        let version_minor = env!("CARGO_PKG_VERSION_MINOR").parse::<u32>()?;
+        let version_patch = env!("CARGO_PKG_VERSION_PATCH").parse::<u32>()?;
```

Comment:
This was intentional to fall back on `0` in case the patch part of the version contains an unparsable int